ui/raidboss/popup-text.js

```diff
@@ -361,354 +366,402 @@ class PopupText {
   }
 
   OnTriggerInternal(trigger, matches) {
-    if (!this.options.AlertsEnabled)
-      return;
-    if ('disabled' in trigger && trigger.disabled)
+    let now = +new Date();
+
+    if (!this._OnTriggerInternal_CheckSuppressed(trigger, now))
       return;
 
     // If using named groups, treat matches.groups as matches
     // so triggers can do things like matches.target.
     if ((matches != undefined) && (matches.groups != undefined))
       matches = matches.groups;
 
-    let now = +new Date();
+    // Set up a helper object so we don't have to throw
+    // a ton of info back and forth between subfunctions
+    let TriggerHelper = this._OnTriggerInternal_GetHelper(trigger, matches, now);
```

Comment:
So should it be camelCase for variables and ProperCase for properties/functions? I'll make a pass through again and change the styling to match, just want to only have to do it once :)

Reply:
camelCase for variables for sure.  I think this is the big one for me, especially because there's a lot of mix of variables named ProperCase and camelCase in this patch, and it's a little jarring to read through.

I think eventually I'd like to be camelCase too, but I think there's a lot of inconsistency there, so do whatever the surrounding code does.  I really just need to fix everything up and have eslint apply these rules.

Reply:
I'm willing to do a sweep of the whole file and update everything, since I can just plug it into the raidemulator redesign to test the changes easily now. So then camelCase everything but the class name itself? (as well as leaving the actual trigger properties etc alone)

Reply:
I would prefer one thing at a time.  Like, new code you're adding, don't add variables with ProperCase.  If you want to come back and update things in another pull request, that seems better.  Sorry for the confusing state things are in.